bindings/rust/lib.rs

```diff
@@ -43,11 +43,17 @@ pub fn language() -> Language {
 /// The source of the Python tree-sitter grammar description.
 pub const GRAMMAR: &'static str = include_str!("../../grammar.js");
 
+/// The syntax highlighting queries for this language.
+pub const HIGHLIGHT_QUERIES: &'static str = include_str!("../../queries/highlights.scm");
```

Comment:
I think we should name these constants with a singular "_QUERY", since they each represent just one query.

Reply:
Ah yes! I had the terminology wrong, I thought each stanza in the file was a query, but looking at the [docs](https://tree-sitter.github.io/tree-sitter/using-parsers#pattern-matching-with-queries) it looks like the file is a "query" and each stanza is a "pattern".  Will update.

Reply:
Done